hw3.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="hw3css.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&family=Playfair+Display&display=swap"
+        rel="stylesheet">
+    <title>Домашка 3</title>
+</head>
+
+<body>
+    <div class="white_box">
+        <p class="willow">Willow Crest</p>
```

Comment:
You should not name classes by content, use generally accepted https://github.com/yoksel/common-words
p.s. here u can use 'title' class for example